blog/2015/03/first-software-carpentry-korea.html

```diff
@@ -0,0 +1,78 @@
+---
+layout: blog
+root: ../../..
+author: Victor (Kwnagchun) Lee
+title: "The First Software Carpentry in Korea"
+date: 2015-03-02
+time: "23:37:00"
+category: ["Workshops", "Korea", "Seoul", "Rapa", "xwMOOC"]
+---
+<!-- start excerpt -->
+<p> Last week, xwMOOC <a href="http://www.xwmooc.net"></a> ran the first <a href="http://statkclee.github.io/2015-02-25-seoul/">Software Carpentry workshop</a> in Seoul. The workshop delivered not only an introduction to Unix Shell, Git and GitHub, Python and SQL but also computer science unplugged, rur-ple, python for informatics and cloud basics. Listening to attendees feedback, attendees concerns such as software business, all about what start-ups should know, how software business environment looks like and so on are provided during three days workshops.
+</p>
+<!-- end excerpt -->
+
+<p> Due to lack of instructors, Victor did teach all the lessons. Unfortunately, Dave, who is supposed to be a helper, could attend the first workshop such that one of attendees, Mr. Seo, became a volunteer and did a wonderful job. Even though there were some software installation problems, Mr. Seo and Victor could overcome some technical difficulties. </p>
```

Comment:
I don't think the bug reporting under the Windows 7 is a good idea. Because all the Windows versions are merging into Windows 10. We'd better use our resources on the more productive ones.